templates/kv-explorer/metadata/snippets/component.txt

```diff
@@ -1,9 +1,12 @@
 [[trigger.http]]
-id = "trigger-kv-explorer"
-component = "kv-explorer"
+component = "{{project-name | kebab_case}}"
 route = "/internal/kv-explorer/..."
 
-[component.kv-explorer]
-source = { url = "https://github.com/fermyon/spin-kv-explorer/releases/download/v0.9.0/spin-kv-explorer.wasm", digest = "sha256:07f5f0b8514c14ae5830af0f21674fd28befee33cd7ca58bc0a68103829f2f9c" }
+[component.{{project-name | kebab_case}}]
+source = { url = "https://github.com/fermyon/spin-kv-explorer/releases/foo/v0.9.0/spin-kv-explorer.wasm", digest = "sha256:07f5f0b8514c14ae5830af0f21674fd28befee33cd7ca58bc0a68103829f2f9c" }
```

Comment:
`foo`?

Reply:
Great eye! This was left over from some testing.